### ISSUE TITLE
the reported accuracy

### ISSUE THREAD

**Issue by lslynf** [braindecode/braindecode#165]
In the paper, the Shallow ConvNet can achieve  68 + 5.7 accuracy, but I run the plot_bcic_iv_2a_moabb_cropped.py file for each subject_id and then get the average accuracy, I can't get the same accuracy reported in paper, so can you report the training epoch and other hyperparameter? Thanks a lot!

**Comment by robintibor** [comment]
For the way it was run in original paper, there is code using an older braindecode here: https://github.com/TNTLFreiburg/braindecode/blob/master/examples/bcic_iv_2a_cropped.py 
In general, it should be expected that in new braindecode with new hyperparameters for cropped training, accuracies are at least as good or higher, what did you get?
